packages/html/src/window/tests/window.tsx

```diff
@@ -1,8 +1,9 @@
-import { Window } from '../../window';
+import { WindowNormal } from '..';
 
 const style = `
     .k-window {
         position: relative;
+        width: 300px;
```

Comment:
Maybe we could increase this width?

Reply:
I decided to remove the width limitation completely as it was and leave it to scale according to the content it has. **WDYT**.